src/nvim/ui.c

```diff
@@ -722,6 +715,9 @@ void ui_call_event(char *name, Array args)
   map_foreach_value(&ui_event_cbs, event_cb, {
     Error err = ERROR_INIT;
     Object res = nlua_call_ref(event_cb->cb, name, args, kRetNilBool, NULL, &err);
+    // TODO(bfredl/luukvbaal): should this be documented or reconsidered?
+    // Why does truthy return from Lua callback mean remote UI should not receive
+    // the event.
     if (LUARET_TRUTHY(res)) {
```

Comment:
I assume this was in analogy w/ `nvim_create_autocmd` https://github.com/neovim/neovim/blob/0b8a72b73934d33a05e20c255298e88cd921df32/runtime/doc/api.txt#L3455-L3456

though I don't know what the actual use-case is, here.